go/vt/worker/legacy_split_clone.go

```diff
@@ -567,7 +567,7 @@ func (scw *LegacySplitCloneWorker) copy(ctx context.Context) error {
 
 			for _, c := range chunks {
 				sourceWaitGroup.Add(1)
-				go func(td *tabletmanagerdatapb.TableDefinition, tableIndex int, chunk chunk) {
+				go func(td *tabletmanagerdatapb.TableDefinition, shardIdx, tableIndex int, chunk chunk) {
```

Comment:
nit: Can you please the new parameter "shardIndex"?

For `tableIndex` we reuse the same name as well. This should work?